packages/vscode-extension/package.json

```diff
@@ -604,15 +604,11 @@
       },
       {
         "command": "fx-extension.migrateTeamsManifest",
-        "title": "%teamstoolkit.commands.migrateManifest.title%",
-        "enablement": "fx-extension.isTeamsFx && isWorkspaceTrusted",
```

Comment:
Please keep the `isWorkspaceTrusted` condition and "category" property, otherwise the prefix Teams will be missing. isWorkspaceTrusted is suggested to preserve because extension will read content of workspace file, if user doesn't trust it, this may contain malicious codes.

Reply:
Thanks for the explanation. Updated.